static/js/people.js

```diff
@@ -1289,7 +1289,7 @@ function safe_lower(s) {
 }
 
 export function matches_user_settings_search(person, value) {
-    const email = settings_data.email_for_user_settings(person);
+    const email = person.delivery_email;
 
     return safe_lower(person.full_name).includes(value) || safe_lower(email).includes(value);
```

Comment:
Hmm, this conditional seems like it won't enjoy `delivery_email` being `null`?

Reply:
This works because `safe_lower` takes care of this by using empty string for such case.

Reply:
OK; I'm not sure I like that from a readability standpoint, but it's a preexisting quirk of this code path, and this PR is already complex enough.